internal/rbd/nodeserver.go

```diff
@@ -226,14 +226,14 @@ func populateRbdVol(
 		}
 	}
 	if krbdSupported {
-		rv.MapOptions = req.GetVolumeContext()["mapOptions"]
-		rv.UnmapOptions = req.GetVolumeContext()["unmapOptions"]
 		rv.Mounter = req.GetVolumeContext()["mounter"]
 	} else {
 		// fallback to rbd-nbd,
 		// ignore the mapOptions and unmapOptions as they are meant for krbd use.
```

Comment:
Please update/correct this comment too.

Reply:
Not really! I will refresh the series in such a way that the comments make sense.